modules/using-an-algorithm/pages/index.adoc

```diff
@@ -29,6 +31,9 @@ See xref:gui:admin-portal:gsql-output-file.adoc[] for details.
 
 == Packaged template queries
 
+If your version of TigerGraph Server is 3.8 or higher, you can use packaged queries to run most Graph Data Science Library algorithms.
+Packaged queries cannot be run from GraphStudio.
```

Comment:
I would move this sentence after the following two sentences. That is, first explain how a user CAN run packaged queries. Then mention that they can't use GraphStudio.